MentoAD.sln

```diff
@@ -15,6 +15,30 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Task4", "MentoringHomeWork\
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Task6", "MentoringHomeWork\HW4\Task6\Task6.csproj", "{55CFB186-E84A-44AF-A627-E1A4058DB3DA}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HW.05.Task1", "MentoringHomeWork\HW5\HW.05.Task1\HW.05.Task1.csproj", "{0663FD29-10AB-42C1-A186-B2718B72E311}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GoodDay", "MentoringHomeWork\HW5\GoodDay\GoodDay.csproj", "{1BA259E5-999C-42C4-9170-CA0F1C38D537}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "LattersCount", "MentoringHomeWork\HW5\LattersCount\LattersCount.csproj", "{83EF544B-7064-46CC-AFB4-82D7FDD35AB3}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HW.05.Task2", "MentoringHomeWork\HW5\HW.05.Task2\HW.05.Task2.csproj", "{70EB9210-8462-4551-A70C-86712BA222F2}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HW.05.Task3", "MentoringHomeWork\HW5\HW.05.Task3\HW.05.Task3.csproj", "{B4A144F5-73AF-457D-A60B-9FC92E34EBEB}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HW.05.Task4", "MentoringHomeWork\HW5\HW.05.Task4\HW.05.Task4.csproj", "{6E9CF6C8-28E1-4D1A-9D37-1320B0269B3D}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Home Work 5", "Home Work 5", "{00726010-3262-4CE1-9F31-A18E2F921EF5}"
```

Comment:
Please let's use pattern - "HomeWork5" - it;s more Dev style and can be efficient in CI/Cd in the future

Reply:
resolved